3rdparty/CMakeLists.txt

```diff
@@ -267,6 +267,22 @@ add_library(spirv_cross OBJECT
 target_compile_definitions(spirv_cross PUBLIC SPIRV_CROSS_EXCEPTIONS_TO_ASSERTIONS)
 
 
+add_library(imgui
```

Comment:
this needs a `NBL_BUILD_WITH` flag like we do for all the extensions like Radeon Rays